### ISSUE TITLE
One Piece episodes not playing

### ISSUE THREAD

**Issue by diamonddudetcg** [pystardust/ani-cli#1137]
**Metadata (please complete the following information)**
Version: 4.3
OS: Windows 10
Shell: Windows Terminal
Anime: One Piece

**Describe the bug**
Every episode I try to watch says "Episode not released!" and ani-cli instantly closes. This was working yesterday and it's no longer working anymore. I am fairly sure Episode 45 of One Piece **has** been released.

**Steps To Reproduce**

1. Run ani-cli
2. Search One Piece
3. Pick any episode

**Expected behavior**
Episodes should be shown as released and should start playing.

**Screenshots (if applicable; you can just drag the image onto github)**
![image_2023-06-02_053251805](https://github.com/pystardust/ani-cli/assets/1720747/a052d865-fad3-4c91-bfcb-8d2bbca68c6b)
![image](https://github.com/pystardust/ani-cli/assets/1720747/1c9a9eec-5d1a-43dd-8949-cc752ea35117)
![image](https://github.com/pystardust/ani-cli/assets/1720747/1ec05d1e-a06b-42f0-9986-a43afb926741)
![image](https://github.com/pystardust/ani-cli/assets/1720747/25f0b2bc-c148-4cff-b47c-2361df31a3dc)

**Comment by Benjamin5678** [comment]
I just installed this today and it was working fine for a while but then suddenly I was having this issue. I am also on windows but using terminal preview with git bash. Is it possible this issue does not originate locally but with whatever online services the script uses? I tried debugging (bear in mind I don't know what I am doing) and It seems the issue is that in the function "get_episode_url" the variable "resp" from curl is being left blank.
